### ISSUE TITLE
complianceChecker.py returns false error for not recognizing the UsdTransform2d node

### ISSUE THREAD

**Issue by kcoley** [PixarAnimationStudios/USD#1624]
### Description of Issue
`UsdUtils.ComplianceChecker` returns an error when encountering a `UsdTransform2d` Shader prim in a USD

### Steps to Reproduce
1. run UsdUtils.ComplianceChecker on a USD containing a `UsdTransform2d` prim.  Here is a Python snippet as an example, assuming a USDZ called `test.usdz` contains a `UsdTransform2d` prim: 
```python
from pxr import Usd, UsdUtils

checker = UsdUtils.ComplianceChecker(arkit=True, skipARKitRootLayerCheck=False, rootPackageOnly=False, skipVariants=False, verbose=True)
checker.CheckCompliance('test.usdz')

for rule in checker._rules:
    error = rule.__class__.__name__
    failures = rule.GetFailedChecks()
    if len(failures) > 0:
        print(error)
        print(failures)
```

It returns as an error:
`["Shader </test/Looks/usdPreviewSurface1SG/file1/UsdTransform2d> has unsupported info:id 'UsdTransform2d'."]`

I am working on a PR fix for this, coming up.

**Comment by kcoley** [comment]
PR here: https://github.com/PixarAnimationStudios/USD/pull/1625

**Comment by jilliene** [comment]
Filed as internal issue #USD-6905

**Comment by sunyab** [comment]
The fix for this issue was merged as commit 8690742 and was released in 21.11. Closing this out, thanks!
